crates/cargo-contract/src/cmd/extrinsics/mod.rs

```diff
@@ -133,11 +133,15 @@ pub struct ExtrinsicOpts {
     #[clap(long)]
     storage_deposit_limit: Option<BalanceVariant>,
     /// Before submitting a transaction, do not dry-run it via RPC first.
-    #[clap(long)]
+    #[clap(long, conflicts_with = "y")]
     skip_dry_run: bool,
     /// Before submitting a transaction, do not ask the user for confirmation.
-    #[clap(long)]
+    #[clap(long, conflicts_with = "y")]
```

Comment:
Possibly we could change this whole PR just by adding `y` as the short for `skip-confirm`

Reply:
No worries, is great to have your feedback.
This was my initial though too. Do you suggest to come back to the [initial commit](https://github.com/paritytech/cargo-contract/pull/1127/commits/01fc69a7ccba7925e52f9c13212ac0247201cc85#diff-0930d98a0a16379942c5f6a0dc0daabf6ed8c9d1bd0f56749fadaaeb8fc03e46L139).

Reply:
Yes looks good :+1: